# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/performance-timeline/idlharness.any.js.ini
# Commit: 3cd097a7b017
# Full Hash: 3cd097a7b017154cf5f7a2d9ed97c7891d3074f4
# Author: Will Hawkins <whawkins@mozilla.com>
# Date: 2019-04-10 09:58:56
# Regressor Bug: 1539006
# File Overlap Count: 1
# Description:
#   Bug 1539006: Properly implement Performance Timeline Level 2 w3c spec. r=baku,flod
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D26654
# ==============================================================================

diff -r ea2baf2c9ae7 -r 3cd097a7b017 testing/web-platform/meta/performance-timeline/idlharness.any.js.ini
--- a/testing/web-platform/meta/performance-timeline/idlharness.any.js.ini	Tue Apr 09 20:32:54 2019 +0000
+++ b/testing/web-platform/meta/performance-timeline/idlharness.any.js.ini	Wed Apr 10 00:14:39 2019 +0000
@@ -2,7 +2,10 @@
   expected: TIMEOUT
 
 [idlharness.any.serviceworker.html]
-  [PerformanceObserver interface: attribute supportedEntryTypes]
+  [PerformanceObserver interface: calling observe(PerformanceObserverInit) on observer with too few arguments must throw TypeError]
+    expected: FAIL
+  
+  [PerformanceObserver interface: operation observe(PerformanceObserverInit)]
     expected: FAIL
 
   [PerformanceMark interface: attribute detail]
@@ -16,10 +19,13 @@
 
 
 [idlharness.any.sharedworker.html]
-  [PerformanceObserver interface: attribute supportedEntryTypes]
+  [PerformanceMark interface: attribute detail]
     expected: FAIL
 
-  [PerformanceMark interface: attribute detail]
+  [PerformanceObserver interface: calling observe(PerformanceObserverInit) on observer with too few arguments must throw TypeError]
+    expected: FAIL
+  
+  [PerformanceObserver interface: operation observe(PerformanceObserverInit)]
     expected: FAIL
 
   [PerformanceMark interface: mark must inherit property "detail" with the proper type]
@@ -30,10 +36,13 @@
 
 
 [idlharness.any.html]
-  [PerformanceObserver interface: attribute supportedEntryTypes]
+  [PerformanceMark interface: attribute detail]
     expected: FAIL
 
-  [PerformanceMark interface: attribute detail]
+  [PerformanceObserver interface: calling observe(PerformanceObserverInit) on observer with too few arguments must throw TypeError]
+    expected: FAIL
+  
+  [PerformanceObserver interface: operation observe(PerformanceObserverInit)]
     expected: FAIL
 
   [PerformanceMark interface: mark must inherit property "detail" with the proper type]
@@ -44,10 +53,13 @@
 
 
 [idlharness.any.worker.html]
-  [PerformanceObserver interface: attribute supportedEntryTypes]
+  [PerformanceMark interface: attribute detail]
     expected: FAIL
 
-  [PerformanceMark interface: attribute detail]
+  [PerformanceObserver interface: calling observe(PerformanceObserverInit) on observer with too few arguments must throw TypeError]
+    expected: FAIL
+  
+  [PerformanceObserver interface: operation observe(PerformanceObserverInit)]
     expected: FAIL
 
   [PerformanceMark interface: mark must inherit property "detail" with the proper type]